zeta/templates/zeta.template.c

```diff
@@ -239,3 +296,25 @@ void __zt_storage_thread(void)
 }
 
 #endif
+
+#ifdef CONFIG_ZETA_FORWARDER
+static void __zt_forwarder_thread(void)
+{
+    zt_isc_packet_t packet = {0};
+    u32_t packet_counter   = 0;
+    u8_t *data;
+
+    u8_t buffer[50] = {0};
```

Comment:
The buffer seems to be short in order to send the biggest zeta packet size (255).